docs/source/notebooks/07_Making_and_using_hotstart_files.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I think we should promote the "hru" formulation instead of passing area, elevation, latitude and longitude directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>True, good catch. fixed.</p>